Assets/Scripts/Character/Movement/Jump.cs

```diff
@@ -0,0 +1,83 @@
+using UnityEngine;
+
+[RequireComponent(typeof(Rigidbody2D))]
+public class Jump : PlayableBehaviour<Jump.Context>
+{
+    public class Context
+    {
+        public float jumpSpeed;
+
+        public Context(float jumpSpeed)
+        {
+            this.jumpSpeed = jumpSpeed;
+        }
+    }
+
+    private Rigidbody2D rb;
```

Comment:
I prefer full names:
```suggestion
    private new Rigidbody2D rigidbody;
```